lib/item_repository.rb

```diff
@@ -65,12 +71,12 @@ def find_all_by_merchant_id(merchant_id)
 
   def create(attributes)
     id = @items[-1].id.to_i
+
     id += 1
     id = id.to_s
     attributes[:id] = id
```

Comment:
Maybe a helper method here would be appropriate?

Reply:
Agree, we can have a refactoring session soon!